docs/clients/dotnet/5.0/reading/README.md

```diff
@@ -115,54 +115,54 @@ To ensure that the EventStoreDB server follows link events when reading, ensure
 
 The parameters are:
 
-| Parameter | Description |
-|:----------|:------------|
-| `string Stream` | The name of the stream to read |
-| `long start`    | The earliest event to read (inclusive). For the special case of the start of the stream, you should use the constant `StreamPosition.Start`. |
-| `int count` | The maximum number of events to read in this request (assuming that many exist between the start specified and the end of the stream) |
+| Parameter             | Description                                                                                                                                                         |
+| :-------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `string Stream`       | The name of the stream to read                                                                                                                                      |
+| `long start`          | The earliest event to read (inclusive). For the special case of the start of the stream, you should use the constant `StreamPosition.Start`.                        |
+| `int count`           | The maximum number of events to read in this request (assuming that many exist between the start specified and the end of the stream)                               |
 | `bool resolveLinkTos` | Determines whether any link events encountered in the stream will be resolved. See the discussion on [Resolved Events](#resolvedevent) for more information on this |
 
 ## Read a stream backwards
 
-The `ReadStreamEventsBackwardAsync` method reads the requested number of events in the reverse order from that in which they were originally written to the stream from a specified starting point.
+The `ReadStreamEventsBackwardAsync` method reads the requested number of events in the reverse order from that in which they were originally appended to the stream from a specified starting point.
 
 The parameters are:
 
-| Parameter | Description |
-|:----------|:------------|
-| `string Stream` | The name of the stream to read |
-| `long start` | The latest event to read (inclusive). For the end of the stream use the constant `StreamPosition.End` |
-| `int count` | The maximum number of events to read in this request (assuming that many exist between the start specified and the start of the stream) |
+| Parameter             | Description                                                                                                                                                         |
+| :-------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
```

Comment:
We can keep it for now, but I am not a big fan of aligning Markdown tables as they will not be kept this way anyway. I also have wrapping in the editor and this line takes like three lines, hard to work with.

Reply:
I'm using https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one it applies the formatting automatically (can also do based on prettier). But if you prefer, I can revert those changes.

Reply:
@alexeyzimarev I reverted format changes to tables.